src/Symfony/Component/VarDumper/composer.json

```diff
@@ -45,5 +46,10 @@
     "bin": [
         "Resources/bin/var-dump-server"
     ],
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "extra": {
+        "branch-alias": {
+            "dev-main": "5.3-dev"
```

Comment:
should be removed

Reply:
@Nyholm told me to add it. I'm confused now :D How can I make sure that the Travis build uses changes from both packages in the test?

Reply:
it should be removed, there is no main branch in Symfony
but I get you're both looking for a solution for the CI, which is supposed to pass.
I'm having a look right now.

Reply:
It's removed now.